tests/Tests/Properties/Instances.hs

```diff
@@ -43,6 +44,16 @@ tl_mempty         = mempty === (unpackS (mempty :: TL.Text))
 t_IsString        = fromString  `eqP` (T.unpack . fromString)
 tl_IsString       = fromString  `eqP` (TL.unpack . fromString)
 
+t_Binary s        =
+  case decodeOrFail . encode $ T.pack s of
```

Comment:
Please use `instance Arbitrary Text` from `Tests.QuickCheckUtils`, it generates more "interesting" inputs than `T.pack` does. Same below for the lazy one.

Reply:
Done